app/main.py

```diff
@@ -1,8 +1,41 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) \
+            -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: int, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: str) -> float:
+        single_car_wash_price = car.comfort_class * \
+            (self.clean_power - car.clean_mark) * \
```

Comment:
make it more readable. use braces instead

Reply:
I don't quite understand where I can use braces in lines 17-19. You mean like that: 
        single_car_wash_price = (car.comfort_class) * \
            (self.clean_power - car.clean_mark) * \
            (self.average_rating / self.distance_from_city_center)

Reply:
![image](https://user-images.githubusercontent.com/72562232/203027241-baf448ab-688e-4878-84a0-cb9023d14df1.png)
use checklist